src/Parser/Shaman/Elemental/Modules/Talents/ElementalBlast.js

```diff
@@ -14,6 +14,26 @@ class ElementalBlast extends Analyzer {
     combatants: Combatants,
   };
 
+  currentBuffAmount=0;
+  lastFreshApply=0;
+  resultDuration=0;
+
+  on_removebuff(event) {
+    if (event.ability.guid === SPELLS.ELEMENTAL_BLAST_CRIT.id ||event.ability.guid === SPELLS.ELEMENTAL_BLAST_MASTERY.id ||event.ability.guid === SPELLS.ELEMENTAL_BLAST_HASTE.id){
```

Comment:
You can simplify this by making an array of the ids (preferably a constant outside the class)
```jsx
const ELEMENTAL_BLAST_IDS = [
   SPELLS.ELEMENTAL_BLAST_CRIT.id,
   SPELLS.ELEMENTAL_BLAST_MASTERY.id,
   SPELLS.ELEMENTAL_BLAST_HASTE.id,
];
```
and then using `if (ELEMENTAL_BLAST_IDS.includes(event.ability.guid))` (or if you prefer, you can make that a `Set` but it's nearly the same).